src/ch/imedias/rsccfx/model/connectionutils/RunRudp.java

```diff
@@ -317,16 +322,77 @@ private void closeAll() {
   }
 
   private void setupClosables() {
-    for (Field field : this.getClass().getDeclaredFields()) {
```

Comment:
I got a Concurrent modification Exception: for the Rudp proxy we need two threads on both sides, it might have been the case that both of them tried to close everything simultaniously. So quickfix was to revert to the previous version. Feel free to improve your solution, just make sure you test it, so the concurrent mod - error won't occur again :)